docs/lib/analytics/fragments/ios/identifyuser.md

```diff
@@ -17,16 +16,16 @@ func identifyUser() {
 
         print("Got identityId: \(identityId). UserState: \(userState)")
         let location = AnalyticsUserProfile.Location(latitude: 47.606209,
-                                                        longitude: -122.332069,
-                                                        postalCode: "98122",
-                                                        city: "Seattle",
-                                                        region: "WA",
-                                                        country: "USA")
+                                                     longitude: -122.332069,
+                                                     postalCode: "98122",
+                                                     city: "Seattle",
+                                                     region: "WA",
+                                                     country: "USA")
         let properties = ["userState": "\(userState)"]
         let userProfile = AnalyticsUserProfile(name: "name",
-                                                email: "name@email.com",
-                                                location: location,
-                                                properties: properties)
+                                               email: "name@email.com",
```

Comment:
Nit, and I didn't use do this yet, but I intended to use `example.com` in all examples. It's goodly of us to do as it's what [RFC2606](https://tools.ietf.org/rfc/rfc2606.txt) instructs and de rigueur.